solarforecastarbiter/io/api.py

```diff
@@ -15,6 +16,18 @@
 
 
 BASE_URL = 'https://api.solarforecastarbiter.org'
+logger = logging.getLogger(__name__)
+
+
+def request_cli_access_token(user, password):
+    req = requests.post(
+        'https://solarforecastarbiter.auth0.com/oauth/token',
+        data={'grant_type': 'password', 'username': user,
+              'audience': BASE_URL,
+              'password': password,
+              'client_id': 'c16EJo48lbTCQEhqSztGGlmxxxmZ4zX7'})
```

Comment:
I'm guessing this corresponds to something meaningful...

Reply:
yes, but not something private